### PR TITLE
update tigera/api pin to latest (v3.12 branch)

### PR DESCRIPTION
**Pull request by rene-dekker**

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and delete the branch after I've merged it.

**Review by doublek — APPROVED** [review]

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
